src/main/java/com/thebois/views/GameScreen.java

```diff
@@ -35,6 +44,30 @@ public GameScreen(final ShapeRenderer batch) {
         camera = new OrthographicCamera();
         viewport = new FitViewport(worldWidth, worldHeight, camera);
         camera.translate(worldWidth / 2, worldHeight / 2);
+
+        initColony();
+    }
+
+    /**
+     * This method is temporary do not forget to remove!!!.
+     */
+    void initColony() {
+        // Change this variable to spawn a different amount of pawns on the screen.
+        final int numberOfPawns = 1000;
+
+        colony = new Colony();
+
+        for (int i = 0; i < numberOfPawns; i++) {
+            colony.createBeing(new Position(random.nextInt((int) worldWidth * 2),
+                                            random.nextInt((int) worldWidth * 2)));
+        }
+
+        for (IBeing pawn : colony.getBeings()) {
+            pawn.walkTo(new Position(random.nextInt((int) worldWidth),
+                                     random.nextInt((int) worldWidth)));
+        }
```

Comment:
Contrary to what I told you (sorry), this belongs in the constructor of `Colony` instead.

Reply:
Yeah, I was thinking that too, but it was a quick fix and I didn't give it much thought since it was only going to be used for a quick test demo. I will think this over and refactor the code.